elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -72,7 +72,8 @@ export interface P2PClosePacket {
 }
 
 export interface P2PConfig {
-	readonly blacklistedPeers?: ReadonlyArray<P2PPeerInfo>;
+	// tslint:disable-next-line:readonly-keyword
+	blacklistedPeers: P2PPeerInfo[];
```

Comment:
do we need to change it here?
maybe holding instance type should be changed but not the P2P config?